src/k8s/tests/request.test.js

```diff
@@ -300,4 +300,18 @@ describe('request.js', () => {
       expect(vm.spec.template.spec.domain.resources.requests.memory).toBe('1G');
       return vm;
     }));
+  it('every disk has volume', () =>
+    createVM(k8sCreate, templates, basicSettings, networks).then(vm => {
+      vm.spec.template.spec.domain.devices.disks.forEach(disk => {
+        expect(vm.spec.template.spec.volumes.filter(volume => volume.name === disk.volumeName)).toHaveLength(1);
+      });
+      return vm;
+    }));
+  it('every disk has volume - cloud init', () =>
+    createVM(k8sCreate, templates, basicSettingsCloudInit, networks).then(vm => {
```

Comment:
repeated code, maybe make a function out of this?

Reply:
fixed